iree/test/microbenchmarks/BUILD

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Tests compilation of microbenchmarks.
+
+load("//build_tools/bazel:enforce_glob.bzl", "enforce_glob")
+load("//build_tools/bazel:iree_check_test.bzl", "iree_check_single_backend_test_suite")
+
+package(
+    default_visibility = ["//visibility:public"],
+    features = ["layering_check"],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+# Check host features compilation for Linalg microbenchmarks.
+iree_check_single_backend_test_suite(
+    name = "check_linalg_microbenchmarks_compilation",
+    srcs = enforce_glob(
```

Comment:
This `enforce_glob` is going to fail cause it is going to look for the `mhlo_*.mlir` files listed below. The glob basically ensures that `*.mlir` files (what is specified in `include`) is same as what is specifically enumerated in the list. There is going to be a mismatch here. I'd suggest just combining everything into one `iree_check_single_backend_test_suite`. If your input doesnt have `mhlo` ops using the `iree-input-type=mhlo` wont throw an error. So you can include `linalg*.mlir` files as well.